homeassistant/helpers/registry.py

```diff
@@ -11,7 +11,7 @@
     from .storage import Store
 
 SAVE_DELAY = 10
-SAVE_DELAY_STARTING = 300
+SAVE_DELAY_LONG = 300
```

Comment:
```suggestion
SAVE_DELAY_LONG = 180
```

We can drop this as startup should never take more than 180 seconds now unless something is very wrong